pageserver/src/tenant_mgr.rs

```diff
@@ -485,9 +465,11 @@ pub fn list_tenants(remote_index: &RemoteTimelineIndex) -> Vec<TenantInfo> {
         .iter()
         .map(|(id, tenant)| {
             let has_in_progress_downloads = remote_index
-                .tenant_entry(id)
-                .map(|entry| entry.has_in_progress_downloads());
+            .tenant_entry(id)
+            .map(|entry| entry.has_in_progress_downloads());
 
+            // TODO this is not correct when we might have remote storage sync disabled:
```

Comment:
Not really related, but found in the logs that we error when remote storage is disabled, so I've left a comment.